frontend/device/kobo/device.lua

```diff
@@ -119,6 +121,25 @@ local KoboSnow = Kobo:new{
     },
 }
 
+-- Kobo Aura H2O2, Rev2:
+-- FIXME: This needs fixing, at the very least on the touch protocol front, c.f., #3925
+local KoboSnowRev2 = Kobo:new{
+    model = "Kobo_snow",
+    hasFrontlight = yes,
+    touch_snow_protocol = true,
```

Comment:
I'd just remove any touch protocol adjustments so that the proper one can be additive in the future. Besides which, we know this config doesn't work and the default just might. :-P

Reply:
Yeah, I nearly tried to just go with the original H2O config, just for kicks...